pippy/hf/_SaveModule.py

```diff
@@ -41,11 +93,12 @@ def _save_index(
     if not os.path.exists(checkpoint_dir):
         os.mkdir(checkpoint_dir)
 
-    with open(filepath, "w") as f:
-        f.write(json_str)
+    _atomic_write(json_str, filepath)
```

Comment:
To add comment: why do we think atomic write is important here?

Reply:
Maybe accidentally delete the entire _atomic_write?